### PR TITLE
Updated the maps version

### PR DESCRIPTION
**Pull request by cafesilencio**
## Description

Updating the maps SDK version to incorporate recent fixes.  The maps SDK update is expected to incorporate modifications to address https://github.com/mapbox/mapbox-gl-native-internal/issues/268 as well as a crash when starting navigation in an activity with a custom puck.

- [x] I have added any issue links
- [ ] I have added all related labels (`bug`, `feature`, `new API(s)`, `SEMVER`, etc.)
- [ ] I have added the appropriate milestone and project boards

### Goal

Update Nav. SDK with the latest maps offering.

## Testing

Please describe the manual tests that you ran to verify your changes

- [ ] I have tested locally (including `SNAPSHOT` upstream dependencies if needed) through testapp/demo app and run all activities to avoid regressions
- [ ] I have tested via a test drive, or a simulation/mock location app
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have updated the `CHANGELOG` including this PR
- [ ] We might need to update / push `api/current.txt` files after running `$> make 1.0-core-update-api` (Core) / `$> make 1.0-ui-update-api` (UI) if there are changes / errors we're 🆗 with (e.g. `AddedMethod` changes are marked as errors but don't break SemVer) 🚀 If there are SemVer breaking changes add the `SEMVER` label. See [Metalava](https://github.com/mapbox/mapbox-navigation-android/blob/master/docs/metalava.md) docs

### CODE REVIEW

File: libnavigation-ui/build.gradle [hunk 1]
```diff
@@ -76,6 +76,8 @@ dependencies {
 
   // Mapbox Maps SDK
   api dependenciesList.mapboxMapSdk
+  // workaround for a missing transitive artifact
+  implementation "com.mapbox.mapboxsdk:mapbox-android-sdk-gl-core:1.8.1"
```
Comment (kmadsen) [inline libnavigation-ui/build.gradle:80]: Do we have a plan to remove this?

File: libnavigation-ui/src/main/java/com/mapbox/navigation/ui/internal/route/RouteConstants.java [hunk 2]
```diff
@@ -50,6 +50,6 @@
   public static final String MAPBOX_LOCATION_ID = "mapbox-location";
   public static final double MINIMUM_ROUTE_LINE_OFFSET = .001;
   public static final long ROUTE_LINE_VANISH_ANIMATION_DURATION = 1000;
-  public static final long ROUTE_LINE_VANISH_ANIMATION_DELAY = 300;
+  public static final long ROUTE_LINE_VANISH_ANIMATION_DELAY = 350;
```
Comment (Guardiola31337) [inline libnavigation-ui/src/main/java/com/mapbox/navigation/ui/internal/route/RouteConstants.java:53]: Is this needed after bumping to `9.3.0-beta.1`? It seems unrelated to OP

File: libnavigation-ui/build.gradle (hunk 1 above)
Comment (cafesilencio) [reply libnavigation-ui/build.gradle:80]: I simply added back what I saw was there before. I don't know the answer to your question.

Comment (Guardiola31337) [reply libnavigation-ui/build.gradle:80]: cc @LukasPaczos

Comment (kmadsen) [reply libnavigation-ui/build.gradle:80]: Also asking because the comment is "// workaround for a missing transitive artifact"

Does that mean it's temporary? or should it fit in as something like dependenciesList.mapboxGLCore

File: libnavigation-ui/src/main/java/com/mapbox/navigation/ui/internal/route/RouteConstants.java (hunk 2 above)
Comment (cafesilencio) [reply libnavigation-ui/src/main/java/com/mapbox/navigation/ui/internal/route/RouteConstants.java:53]: It's unrelated but I found it to be a better value when testing for the crash. It didn't seem like it would do any harm including it here.

File: libnavigation-ui/build.gradle (hunk 1 above)
Comment (LukasPaczos) [reply libnavigation-ui/build.gradle:80]: This should be resolved, we can remove and retest.

Comment (LukasPaczos) [reply libnavigation-ui/build.gradle:80]: https://github.com/mapbox/mapbox-navigation-android/pull/3312
